.clang-format

```diff
@@ -3,8 +3,12 @@ Standard: Cpp11
 ColumnLimit: 80
```

Comment:
What's the rationale for the changes to this file? It seems to introduce more changes than it preserves. Shouldn't we stick with whatever the defaults of the Google style are?

Reply:
+1